package duke.exceptions;

/**
 * Signals that some task is already done when it tried to mark it as done again
 */
public class TaskDoneException extends Exception{
}
